usagov/vote-gov-drupal
# VOTE-1106: Add translatable external links title

## Jira ticket (required)
[VOTE-1106](https://bixal-projects.atlassian.net/browse/VOTE-1106)

## Description (optional)
Add translatable external links title to provide extra context on hover and for screen readers.

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. lando retune

### Post-deploy steps
1. cd into the theme directory and run npm run build and lando drush cr

### Testing steps
1. Hover over external link to see the text "External link opens in new window"
2. Test other languages that the translation is working (we don't have translation for Yup'ik and Navajo)
